core/src/validations.rs

```diff
@@ -1,13 +1,66 @@
+use crate::utils::get_output_pointer_from_input;
+use std::collections::HashMap;
 use witnet_crypto::hash::Sha256;
 use witnet_crypto::merkle::merkle_tree_root as crypto_merkle_tree_root;
-use witnet_data_structures::chain::{Block, Epoch, Hash, Hashable, Transaction};
+use witnet_data_structures::chain::{
+    Block, Epoch, Hash, Hashable, Output, OutputPointer, Transaction, TransactionsPool,
+};
 
 /// Function to validate block's coinbase
```

Comment:
```suggestion
/// Function to validate block's mint
```

Reply:
Are we using the term `mint` or `coinbase` going forward?

Reply:
Yes, you'r right. We are using `mint`